macros/tables/bigquery/pit.sql

```diff
@@ -66,14 +67,20 @@ pit_records AS (
             {%- endif %}
         {% for satellite in sat_names %}
         {%- set sat_columns = datavault4dbt.source_columns(ref(satellite)) %}
+        {%- if ledts|string|lower in sat_columns|map('lower') %}
         LEFT JOIN {{ ref(satellite) }}
+        {%- else %}
+        LEFT JOIN (
+            SELECT
+                {{ hashkey }},
+                {{ ldts }},
+                COALESCE(LEAD(TIMESTAMP_SUB({{ ldts }}, INTERVAL 1 MICROSECOND)) OVER (PARTITION BY {{ hashkey }} ORDER BY {{ ldts }}),{{ datavault4dbt.string_to_timestamp(timestamp_format, end_of_all_times) }}) AS ledts
```

Comment:
use {{ ledts }}

Reply:
Hi @bschlottfeldt , your requested changes are now committed in this branch. If it works for you, you could approve the Review.